### DISCUSSION TITLE
WoW Talent Tree Manager v1.4.2

### DISCUSSION THREAD

**Discussion by TobiasM95** [TobiasM95/WoW-Talent-Tree-Manager#27] [General]
This is a small update that changes the way icons get handled behind the scenes. This also allows for auto-updating icons in general so that not only the tree structure and talent contents but also the icons will always be up to date.

- Icons get updated automatically from now on
- Icons get shipped in a packed format for better performance
- Small UI improvements

**Full Changelog**: https://github.com/TobiasM95/WoW-Talent-Tree-Manager/compare/v1.4.1...v1.4.2

<hr /><em>This discussion was created from the release <a href='https://github.com/TobiasM95/WoW-Talent-Tree-Manager/releases/tag/v1.4.2'>WoW Talent Tree Manager v1.4.2</a>.</em>

**Comment by drummduck** [comment]
Hey. I was curious how you found out how to convert the trees into an import string. Can you shed any sources on that?

**Reply by drummduck** [reply]
I have a Java project that i'm making for fun to randomize a tree and create a code string. I have tried my best to use you and one other project as a reference and i'm still having an issue. When i try to import on WoWHead it says it's an older version. I feel like there is some sort of update to the export code creation.
